### ISSUE TITLE
Add "Submitted" state

### ISSUE THREAD

**Issue by rperez89** [1Hive/token-request-app#35]
Currently, we have 4 states for the request which are : "Pending" once the request is created, "Approved" once the request is approved, "Expired" once time to expiry reach the date, and "Withdrawed" once the user has withdrawn the funds.

But once that you submit the request and the vote is created we have no way to show to the user a "Submitted" state as the submit action only creates the vote and does not interact with the smart contract in order to get a SubmittedEvent and update the state.

**Comment by rperez89** [comment]
No longer needed
